user_guide_src/source/general/configuration.rst

```diff
@@ -1,125 +1,151 @@
-################################
-Working With Configuration Files
-################################
+#############
+Configuration
+#############
+
+Every framework uses configuration files to define numerous parameters and 
+initial settings. CodeIgniter configuration files define simple classes where 
+the required settings are public properties.  
 
-Every application needs a way to define various settings that affect the application.
-These are handled through configuration files. Configuration files simply
-hold a class that contains its settings as public properties. Unlike in many other frameworks,
-there is no single class that you need to use to access your settings. Instead, you simply
-create an instance of the class and all your settings are there for you.
+Unlike many other frameworks, CodeIgniter configurable items aren't contained in 
+a single file. Instead, each class that needs configurable items will have a 
+configuration file with the same name as the class that uses it. You will find 
+the application configuration files in the **/app/Config** folder.
 
 .. contents::
     :local:
     :depth: 2
 
-Accessing Config Files
-======================
+Working With Configuration Files
+================================
 
-You can access config files within your classes by creating a new instance or using the config function. All of the properties
-are public, so you access the settings like any other property::
+You can access configuration files for your classes in several different ways.
 
-	// Creating new class by hand
+- By using the ``new`` keyword to create an instance::
+ 
+	// Creating new configuration object by hand
 	$config = new \Config\Pager();
 
-	// Creating new class with config function
-	$config = config( 'Pager', false );
+- By using the ``config()`` function::
+
+	// Creating a new object with config function
+	$config = config('Pager', false);
 
 	// Get shared instance with config function
-	$config = config( 'Pager' );
+	$config = config('Pager');
 
 	// Access config class with namespace
 	$config = config( 'Config\\Pager' );
 
-	// Access settings as class properties
+- By using ``\Config\Services::class`` which will apply an instance of the class's config for you::
+
```

Comment:
Services are different than config files. Please remove the sections about services here.

Reply:
Yes, but when you use Services the associated config is (always?) instantiated and applied for you. That was the point I was trying to make. 
Not germane?

Reply:
I think it is convenient but I had to read it a few times to realize what it meant. I think it’s better left out of this part but could maybe have a life as a bonus “note” in Services?